### DISCUSSION TITLE
Langflow support

### DISCUSSION THREAD

**Discussion by hakan-77** [milvus-io/milvus#34465] [Ideas & Feature requests]
[Langflow](https://github.com/langflow-ai/langflow/) is an open source to design LangChain with a nice UI. It is gaining a lot of steam and already support many vector databases, but not Milvus.

It would be fantastic to have Milvus support. Code for other vector databases did not seem overly complex, so I hope someone can give it a shot.

**Comment by xiaofan-luan** [comment]
thanks for the suggestion.
@jaelgu please help on it.

**Reply by jaelgu** [reply]
PR submitted: https://github.com/langflow-ai/langflow/pull/2634

**Reply by hakan-77** [reply]
Wow, that was fast @jaelgu @xiaofan-luan Thank you.
